src/twisted/conch/scripts/ckeygen.py

```diff
@@ -311,6 +322,8 @@ def _saveKey(key, options):
     keyTypeName = KeyTypeMapping[key.type()]
     if not options["filename"]:
         defaultPath = os.path.expanduser(f"~/.ssh/id_{keyTypeName}")
+        if platform.system() == "Windows":
+            defaultPath = os.path.expanduser(fr"~\.ssh\id_{keyTypeName}")
```

Comment:
Do we have tests for this ?

Reply:
The tests that cover this are platform-independent, and should pass whether run on Linux or Windows. It's not going to cover both branches in any given run, as a result.

Reply:
This should be covered by existing tests when run on a windows image, it just changes the default to a path appropriate for Windows.